src/constants.py

```diff
@@ -8,6 +8,7 @@
 
 # Chat API
 CHAT_CAPABILITIES_KEY = 'supports_chat_api'
+DEFAULT_CHAT_MODEL_NAME = 'datarobot-deployed-llm'
```

Comment:
Do we want to mention what this is in the readme?

Reply:
added to readme: https://github.com/datarobot-oss/qa-app-streamlit/pull/105/commits/ccaf56bf12a2f2e4bf77f869e138dd40e97b2ab8